lib/inputguess.rb

```diff
@@ -0,0 +1,44 @@
+class InputGuess
+  attr_reader :user_input, :valid_guess
+
+  def initialize
+    @valid_guess = false
+    @user_input = ""
+  end
+
+  def input_guess
```

Comment:
Consider renaming this to something that isn't so similar to the name of your class to make things more clear.